lib/irb/init.rb

```diff
@@ -358,7 +358,7 @@ def IRB.run_config
     if @CONF[:RC]
       begin
         load rc_file
-      rescue LoadError, Errno::ENOENT
+      rescue Errno::ENOENT
       rescue # StandardError, ScriptError
         print "load error: #{rc_file}\n"
```

Comment:
I imagine that the commented-out `# StandardError, ScriptError` would originally be meant to rescue a LoadError in addition to other "standard" exceptions.
(Note that LoadError is not a descendant of StandardError but a descendant of ScriptError)
I think we should simply make the comment work. Otherwise, we can not rescue a LoadError.

Besides, as the code will be going to rescue both StandardError and ScriptError, the message print "**load** error:" is not appropriate here. We can also simply remove the word "load".

```suggestion
      rescue StandardError, ScriptError
        print "error: #{rc_file}\n"
```